pipeline_steps/common.groovy

```diff
@@ -3,7 +3,8 @@ import groovy.json.JsonOutput
 
 // Install ansible on a jenkins slave
 def install_ansible(){
-  sh """#!/bin/bash
+  sh """#!/bin/bash -xe
+    cd ${env.WORKSPACE}
```

Comment:
Yes, I moved the venv to the root of the workspace in the ARA patch, but that got reverted so will have to pull that part of that PR into this one.